cli/log.c

```diff
@@ -10,9 +10,9 @@
 #include <errno.h>
 #include <unistd.h>
 
-bool stdin_isatty;
-bool stdout_isatty;
-bool stderr_isatty;
+static bool stdin_isatty;
```

Comment:
This seems unrelated.

Reply:
Completely unrelated. I have split into another commit.